hack/install-router.sh

```diff
@@ -1,34 +1,71 @@
 #!/bin/bash
```

Comment:
I kept wanting a template, but that requires talking to the server, which was previously optional here, and I tried to preserve. What I really want is a security context for the secrets, and a service telling me the address and root certs of the master. If wishes were fishes...